pkgs/development/tools/rust/maturin/default.nix

```diff
@@ -4,7 +4,7 @@
 let
   inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
Normally Security belongs into inputs and is inherited from all packages.

Reply:
Fixed. Also reformatted the derivation with `nixpkgs-fmt` and removed the `gmp` and `openssl` buildInputs, which are not used anymore.